cli/cmd/install.go

```diff
@@ -345,6 +363,30 @@ func (options *installOptions) validateAndBuild() (*installValues, *pb.All, erro
 		Identity:     identityValues,
 	}
 
+	if options.highAvailability {
```

Comment:
Looking forward to the future, you *might* want to have a function in the chart that configures this. Definitely not a blocker as that can happen later =)

Reply:
yeah, we should plot out a better factoring for all of this, but i'd rather do that after we have a basic upgrade on master

Reply:
+1 from me, just wanted folks to be aware.